Simplified/Reader2/LCP/LCPLibraryService.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  LCPLibraryService.swift
+//  r2-testapp-swift
+//
+//  Created by Mickaël Menu on 01.02.19.
+//
+//  Copyright 2019 Readium Foundation. All rights reserved.
+//  Use of this source code is governed by a BSD-style license which is detailed
+//  in the LICENSE file present in the project repository where this source code is maintained.
+//
+
+#if LCP
+
+import Foundation
+import UIKit
+import R2Shared
+import ReadiumLCP
+
+
+@objc class LCPLibraryService: NSObject, DRMLibraryService {
```

Comment:
a overall description of how this class works (like which method is supposed to be called when, meaning in case we have bugs) would be good here I think.

Reply:
This probably can be better explained in wiki, this service is a similar to Adobe DRM and is based on DRMLibraryService protocol. We already have an article about Adobe DRM which needs an update to reflect recent changes, I'll add LCP DRM there as well.